auth-validation.js

```diff
@@ -0,0 +1,142 @@
+// 비밀번호 관련 기능 모듈 가져오기
+import { pw } from "./toggle-pw.js";
+
+// DOM 요소 참조
+const form = document.querySelector(".js-form");
+const input = document.querySelectorAll(".js-input");
+const button = document.querySelector(".js-button");
+const emailError = document.querySelector(".js-email-error");
+const pwError = document.querySelector(".js-pw-error");
+const nicknameError = document.querySelector(".js-nickname-error");
```

Comment:
P2:
해당 변수도 errorMessage가 들어갈 요소이니 아래와 같은 이름이 더 유의미할 것 같습니다.

```suggestion
const nicknameErrorContainer = document.querySelector(".js-nickname-error");
```